HCCGo/app/js/clusterLandingCtrl.js

```diff
@@ -111,7 +111,7 @@ clusterLandingModule.controller('clusterLandingCtrl', ['$scope', '$log', '$timeo
   });
 
 
-  $scope.refreshCluster = function(force) {
+  $scope.refreshCluster = function(force=true) {
```

Comment:
for some reason, it gets undefined as the value sometimes from the scope, even though I checked all the values going in.